rest/config_test.go

```diff
@@ -1812,16 +1815,9 @@ func TestSetupDbConfigWithSyncFunction(t *testing.T) {
 					RemoteConfigTlsSkipVerify: true,
 				}
 			}
-			if test.errExpected != nil {
-				test.errExpected = &JavaScriptLoadError{
-					JSLoadType: SyncFunction,
-					Path:       sync,
-					Err:        test.errExpected,
-				}
-			}
 			err := dbConfig.setup(dbConfig.Name, BootstrapConfig{}, nil, nil, false)
 			if test.errExpected != nil {
-				require.True(t, errors.As(err, &test.errExpected)) // nolint: govet // CBG-2242
+				assertX509UnknownAuthority(t, err, test.errExpected)
 			} else {
 				assert.Equal(t, test.jsSyncFnExpected, *dbConfig.Sync)
```

Comment:
as mentioned in previous comment, do the `require.NoError` in these other else blocks